index.js

```diff
@@ -12,6 +12,10 @@ import {
   createElement as h,
 } from "./react-deps.js";
 
+/**
+ * consts
+ */
+export { eventsHistory } from "./use-location";
```

Comment:
While it seems convenient to export these, I really would not want to export implementation-specific location constants from `wouter`. The reason is that wouter can be customised with a variety of locations (hash-based, state-based, memory, static location for SSR and so on). Let's just make that `use-location.js` specific (which by the way should be named `use-browser-location.js` since it's really confusing with `useLocation` hook from `wouter`).

```js
import { events } from "wouter/use-location";

events.forEach(event => window.addEventListener(event, cb));
```

Reply:
thank you, fixed it as you requested